test/integration/e2e_test.go

```diff
@@ -292,14 +292,17 @@ func TestE2EFullIntegration(t *testing.T) {
 	require.NotNil(t, token)
 
 	idTokenClaims := token.IDToken.Claims
-	username := idTokenClaims[oidc.DownstreamUsernameClaim].(string)
-	groups, _ := idTokenClaims[oidc.DownstreamGroupsClaim].([]string)
+	require.Equal(t, env.SupervisorTestUpstream.Username, idTokenClaims[oidc.DownstreamUsernameClaim])
 
-	require.Equal(t, env.SupervisorTestUpstream.Username, username)
 	if len(env.SupervisorTestUpstream.ExpectedGroups) == 0 {
 		// We only put a groups claim in our downstream ID token if we got groups from the upstream.
-		require.Nil(t, groups)
+		require.Nil(t, idTokenClaims[oidc.DownstreamGroupsClaim])
```

Comment:
Uh shouldn't this be `[]interface{}` instead of `nil`?

Reply:
Good question, I can see that it's `"groups": []` in the raw JWT, but I think maybe something about roundtripping through the session cache leaves it as nil. I'll double check.

Reply:
Oh, the tests failed to tell me you were right and I broke the test in the dex case 🤦🏻 ... fixing:

```
    e2e_test.go:277: second kubectl command took 1.019502981s
    e2e_test.go:299: 
        	Error Trace:	e2e_test.go:299
        	Error:      	Expected nil, but got: []interface {}{}
        	Test:       	TestE2EFullIntegration
```